pwiz_tools/Skyline/FileUI/PeptideSearch/SearchSettingsControl.cs

```diff
@@ -247,13 +258,16 @@ public void InitializeEngine()
 
         private void LoadComboboxEntries()
         {
-            LoadFragmentIonEntries();
-            LoadMs2AnalyzerEntries();
+            if (!ImportPeptideSearch.IsFeatureDetection)

```

Comment:
Less code churn and indentation if you invert this if (if feature detection, return early).

Reply:
Thanks, I'll take a look

Reply:
yeah, better thanks